packages/react/src/feature-link-list/FeatureLinkList.tsx

```diff
@@ -0,0 +1,24 @@
+import { Stack } from '../stack';
+import {
+	FeatureLinkListItem,
+	FeatureLinkListItemProps,
+} from './FeatureLinkListItem';
+import { FeatureLinkListBackground } from './utils';
+
+export type FeatureLinkListProps = {
+	links: Omit<FeatureLinkListItemProps, 'children'>[];
+	background?: FeatureLinkListBackground;
+};
+
+export const FeatureLinkList = ({
+	links,
+	background,
+}: FeatureLinkListProps) => {
+	return (
+		<Stack as="ul" borderTop>
+			{links.map(({ ...props }, index) => (
```

Comment:
Shouldn't need to spread the `props` here:

```suggestion
			{links.map((props, index) => (
```